lib/src/app/app.dart

```diff
@@ -16,25 +17,151 @@ class AppClient {
     return response.organizations;
   }
 
+  /// Get all [OrganizationIdentity]s that have access to a [Location].
+  Future<List<OrganizationIdentity>> getOrganizationsWithAccessToLocation(String locationId) async {
+    final request = GetOrganizationsWithAccessToLocationRequest();
+    final GetOrganizationsWithAccessToLocationResponse response = await _client.getOrganizationsWithAccessToLocation(request);
+    return response.organizationIdentities;
+  }
+
+  /// List the [Organization]s a user belongs to
+  Future<List<OrgDetails>> listOrganizationsByUser(String userId) async {
+    final request = ListOrganizationsByUserRequest()..userId = userId;
+    final ListOrganizationsByUserResponse response = await _client.listOrganizationsByUser(request);
+    return response.orgs;
+  }
+
   /// Get a specific [Organization] by ID
   Future<Organization> getOrganization(String organizationId) async {
     final getOrganizationRequest = GetOrganizationRequest()..organizationId = organizationId;
     final GetOrganizationResponse response = await _client.getOrganization(getOrganizationRequest);
     return response.organization;
   }
 
-  /// List the [Location] of a specific [Organization] that the currently authenticated user has access to
-  Future<List<Location>> listLocations(Organization organization) async {
-    final listLocationsRequest = ListLocationsRequest()..organizationId = organization.id;
-    final ListLocationsResponse response = await _client.listLocations(listLocationsRequest);
-    return response.locations;
+  /// Checks for namespace availablity throughout all [Organization]s.
+  Future<bool> getOrganizationNamespaceAvailability(String publicNamespace) async {
+    final request = GetOrganizationNamespaceAvailabilityRequest()..publicNamespace = publicNamespace;
+    final GetOrganizationNamespaceAvailabilityResponse response = await _client.getOrganizationNamespaceAvailability(request);
+    return response.available;
+  }
+
+  /// Update an [Organization]
+  Future<Organization> updateOrganization(String organizationId,
+      {String? name, String? publicNamespace, String? region, String? cid}) async {
+    final request = UpdateOrganizationRequest()
+      ..organizationId = organizationId
+      ..name = name ?? ''
+      ..publicNamespace = publicNamespace ?? ''
+      ..region = region ?? ''
+      ..cid = cid ?? '';
+    final UpdateOrganizationResponse response = await _client.updateOrganization(request);
+    return response.organization;
+  }
+
+  /// Delete an [Organization]
+  Future<void> deleteOrganization(String organizationId) async {
+    final request = DeleteOrganizationRequest()..organizationId = organizationId;
+    await _client.deleteOrganization(request);
+  }
+
+  /// List the members and pending invites for an [Organization].
+  Future<ListOrganizationMembersResponse> listOrganizationMembers(Organization org) async {
+    final request = ListOrganizationMembersRequest()..organizationId = org.id;
```

Comment:
We only care about the `org` argument here for its id. Passing a string is a lot less work than creating and passing a whole `Organization` object, can the param just be a string instead?

Reply:
That was an original function. I didn't change this.

Reply:
Ahh okay, that makes sense! I didn't realize 😅 I think if we can make the change easily that's a great flyby :)